bytecode/cfg.py

```diff
@@ -140,7 +141,7 @@ def update_size(delta, size, maxsize):
         # jump is taken.
         if instr.has_jump():
             taken_size, maxsize = update_size(
-                instr.stack_effect(jump=True), size, maxsize
+                *instr.pre_and_post_stack_effect(jump=True), size, maxsize
```

Comment:
It could be nice while we stabilize this to have a way to bypass this in case we get the computation wrong (using an extra keyword to` _compute_stack_size`)

Reply:
I've made it a kw only argument :)